addons/mail/static/src/models/partner.js

```diff
@@ -340,10 +340,16 @@ registerModel({
         },
         /**
          * @private
-         * @returns {string|undefined}
+         * @returns {string|FieldCommand}
          */
         _computeDisplayName() {
-            return this.display_name || this.user && this.user.display_name;
+            if (this.display_name) {
+                return this.display_name;
+            }
+            if (this.user) {
+                return this.user.displayName;
+            }
+            return clear();
```

Comment:
except the rename of `display_name` to `displayName` for user, the diff in this method is either unnecessary or not going far enough (the clear case was badly handled before, but if you want to make it work correctly, it should also be done if `user.displayName` is undefined)